src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,108 @@
+package duke;
+
+import java.util.ArrayList;
+
+/**
+ * Encapsulates a list of a {@link Task}
+ */
+public class TaskList {
+    private final ArrayList<Task> todo;
```

Comment:
Perhaps the variable name may be changed to more accurately represent that this is a collection of Tasks. May I suggest todoList?